main.go

```diff
@@ -31,6 +31,7 @@ import (
 var (
 	namespace  = flag.String("namespace", "cattle-rancheros-operator-system", "Namespace of the pod")
 	namespaces = flag.String("namespaces", "", "A comma separated list of namespaces to watch")
+	interval   = flag.Int("sync-interval", 10, "Interval (in seconds) for the upgrade channel ticker")
```

Comment:
I'd go here for a string and use `time.ParseDuration`, which directly returns a duration. In this way the user can supply things like "120m", "10s", etc.

Reply:
Not sure how this matches with customer's "maintenance windows", which might happen once a week / month ?!
Do we have documentation how this fits into the larger "image update" workflow ?

Reply:
In addition I would use a default that is _suitable_ in a real environment (1 or 2h maybe?) and shorten it for our tests.

Reply:
That parseduration sounds pretty good, I didn't knew that.

Reply:
wait, so we want this to be exposed or not?? Because if its in the chart values, then its customizable???

Reply:
I updated to expose it in the chart, not sure what the confusion is here to be honest